types/consensus_data.go

```diff
@@ -185,7 +185,7 @@ func (ci *ConsensusData) GetAttestationData() (*phase0.AttestationData, error) {
 }
 
 // GetBlockData ISSUE 221: GetBlockData/GetBlindedBlockData return versioned block only
-func (ci *ConsensusData) GetBlockData() (*api.VersionedProposal, ssz.HashRoot, error) {
+func (ci *ConsensusData) GetBlockData() (blk *api.VersionedProposal, signingRoot ssz.HashRoot, err error) {
```

Comment:
the impl depends on this in anyway or you just changed because you thought it is nice?

Reply:
Impl does not depend. We got confused by the function return values and decided to clarify it